src/ast/elf_parser.cpp

```diff
@@ -1,5 +1,8 @@
 #include "elf_parser.h"
 #include <elf.h>
+#ifndef EM_BPF
+#define EM_BPF 247 // Could be in linux/elf.h, but it can cause conflicts
+#endif
```

Comment:
Why is this necessary?

Reply:
The older systems that I test on don't define it. It only used in the ELF parser to ensure that the Elf Machine is BPF. I was thinking it could be added as a related change, but I can also submit it separately. It just makes it easier for me to build the master branch without having to patch this file.
```c
+#ifndef EM_BPF // EM_BPF may not be defined in older containers, e.g. RHEL7
```

Reply:
I see, makes sense.